freqtrade/freqtradebot.py

```diff
@@ -443,6 +450,52 @@ def create_trade(self, pair: str) -> bool:
         else:
             return False
 
+#
+# BUY / increase positions / DCA logic and methods
+#
+    def process_open_trade_positions(self):
+        """
+        Tries to execute additional buy or sell orders for open trades (positions)
+        """
+        # Walk through each pair and check if it needs changes
+        for trade in Trade.get_open_trades():
+            try:
+                self.adjust_trade_position(trade)
+            except DependencyException as exception:
+                logger.warning('Unable to adjust position of trade for %s: %s',
+                               trade.pair, exception)
+
+    def adjust_trade_position(self, trade: Trade):
+        """
+        Check the implemented trading strategy for adjustment command.
+        If the strategy triggers the adjustment, a new order gets issued.
+        Once that completes, the existing trade is modified to match new data.
+        """
+        for order in trade.orders:
+            if order.ft_is_open:
+                return
+
+        logger.debug(f"adjust_trade_position for pair {trade.pair}")
+        sell_rate = self.exchange.get_rate(trade.pair, refresh=True, side="sell")
```

Comment:
I think side should be "buy" here? you're looking to buy after all ...

Reply:
I've converted this to buy side as that does make more sense overall. Having both seems wrong at this point as supporting negative adjustments / selling is currently way too far future and we have **kwargs if we ever do want to add those. Also the current naming and logic follows all the other callbacks where current_rate and current_profit indicate buying side price and profit.